llvm/test/MC/RISCV/xqcili-linker-relaxation.s

```diff
@@ -0,0 +1,43 @@
+
+# RUN: llvm-mc --triple=riscv32 -mattr=+relax,+experimental-xqcili \
+# RUN:    %s -filetype=obj -o - -riscv-add-build-attributes \
+# RUN:    | llvm-objdump -dr -M no-aliases - \
+# RUN:    | FileCheck %s
+
+## This tests that we correctly emit relocations for linker relaxation when
+## emitting `QC.E.LI` and `QC.LI`.
+
+  .section .text.ex1, "ax", @progbits
+  .global ex1
```

Comment:
we don't need a global symbol at offset 0 for the anchor. if you omit it, you can use `<.text.ex1>`